plug.ts

```diff
@@ -74,9 +79,17 @@ export async function download(options: Options): Promise<string> {
 
   let url;
   if ("urls" in options) {
-    url = options.urls[os];
+    const _url = options.urls[os];
+    if (typeof _url === "string") {
+      url = _url;
+    } else if (_url) {
+      url = _url[arch];
+    }
```

Comment:
```suggestion
   	url = options.urls[os];
    if (typeof url !== "string") {
      url = _url[arch];
    }
```

Looks a bit better imo

Reply:
LGTM. thanks! ✅ [`9f91f9f` (#14)](https://github.com/denosaurs/plug/pull/14/commits/9f91f9f4266532aea3489c9feaf6d3690a7a3fdd)